doc/source/how-to-run-flower-using-docker.rst

```diff
@@ -303,36 +306,16 @@ The ``Dockerfile.serverapp`` contains the instructions that assemble the ServerA
   FROM flwr/serverapp:1.8.0
 
   WORKDIR /app
-  COPY requirements.txt .
-  RUN python -m pip install -U --no-cache-dir -r requirements.txt && pyenv rehash
 
-  COPY server.py task.py ./
+  COPY server.py ./
   ENTRYPOINT ["flower-server-app", "server:app"]
 
 In the first two lines, we instruct Docker to use the ServerApp image tagged ``1.8.0`` as a base
 image and set our working directory to ``/app``. The following instructions will now be
-executed in the ``/app`` directory. Next, we install the ServerApp dependencies by copying the
-``requirements.txt`` file into the image and run ``pip install``. In the last two lines,
-we copy the ServerApp code (``server.py`` and ``task.py``) into the image and set the entry
-point to ``flower-server-app`` with the argument ``server:app``. The argument is the object
-reference of the ServerApp (``<module>:<attribute>``) that will be run inside the ServerApp
-container.
-
-.. important::
-
-  If the requirement.txt contains the `flwr <https://pypi.org/project/flwr/>`__ or
-  `flwr-nightly <https://pypi.org/project/flwr-nightly/>`_ package, please ensure the version in
-  requirement.txt matches the docker image version.
-
-  Stable:
-
-  - Docker image: ``serverapp:1.8.0``
-  - requirement.txt: ``flwr[simulation]==1.8.0``
-
-  Nightly:
-
-  - Docker image: ``serverapp:1.9.0.dev20240501``
-  - requirement.txt: ``flwr-nightly[simulation]==1.9.0.dev20240501``
+executed in the ``/app`` directory. In the last two lines, we copy the ServerApp code into the
```

Comment:
..., we copy the `ServerApp` module into the ...

Reply:
Should we call it `ServerApp` or `server.py` to be consistent with change above?

Reply:
You're right @Robert-Steiner! Let's stick to `server.py`. 

> ..., we copy the `server.py` module into the ...